cypress/e2e/initApp.spec.js

```diff
@@ -0,0 +1,92 @@
+describe('The initial load of Testing Playground', () => {
+  it('renders an example', () => {
```

Comment:
I'm not a great fan of `describe`.  Things are clearer and more compact without it. 

```suggestion
it('renders Testing Playground with an example',() => {
```